.gitignore

```diff
@@ -6,3 +6,4 @@ build
 coverage
 dist
 node_modules
+/hack
```

Comment:
does this need to be here

Reply:
strictly no, but it's a folder where I've been keeping files that I don't want to accidentally post into GitHub, even in a feature branch - secrets, data re: specific implementations of benchmarks, and so on

Reply:
this isn't a blocking request, but you can add it to your own personal version of a gitignore
https://docs.microsoft.com/en-us/azure/devops/repos/git/ignore-files?view=azure-devops&tabs=visual-studio#ignore-files-only-on-your-system